Source/Orts.Simulation/Simulation/RollingStocks/SubSystems/PowerSupplies/DieselEngine.cs

```diff
@@ -1039,12 +1039,12 @@ public void Update(float elapsedClockSeconds)
             DemandedThrottlePercent = Math.Max(DemandedThrottlePercent, ReverseThrottleRPMTab[Locomotive.DieselPowerSupply.DieselEngineMinRpm]);
 
             if (Locomotive.Direction == Direction.Reverse)
-                Locomotive.PrevMotiveForceN *= -1f;
+                Locomotive.PrevTractiveForceN *= -1f;
 
             if ((State == DieselEngineState.Running) && (Locomotive.ThrottlePercent > 0))
             {
-                var abstempMotiveForce = Math.Abs(Locomotive.PrevMotiveForceN);
-                OutputPowerW = ( abstempMotiveForce > 0 ? abstempMotiveForce * Locomotive.AbsSpeedMpS : 0) / Locomotive.DieselEngines.NumOfActiveEngines;
+                var abstempTractiveForce = Math.Abs(Locomotive.PrevTractiveForceN);
+                OutputPowerW = ( abstempTractiveForce > 0 ? abstempTractiveForce * Locomotive.AbsSpeedMpS : 0) / Locomotive.DieselEngines.NumOfActiveEngines;
```

Comment:
Also as a general note, any horsepower calculation done using TractiveForceN should use WheelSpeedMpS/AbsWheelSpeedMpS. As we are considering the force going into the wheel, rather than the force going into the rails, the relevant speed is the wheel speed instead of the rail speed.

Reply:
You're right, I changed it also for the HUD